index.bs

```diff
@@ -266,12 +266,12 @@ and *should* be in the [=ascii code point|ASCII range=].
 
 <h4 id="naming-consultation" class="no-num no-toc">Consultation</h4>
 
-Please consult widely on names in your APIs.
+Please consult widely on names in your features.
```

Comment:
This is about API naming and should be reverted to talk about API rather than features.